data/mods/aftershock_exoplanet/npcs/cyborg_npcs/item_groups_cyborg.json

```diff
@@ -131,7 +131,7 @@
       [ "mp3", 18 ],
       [ "portable_game", 12 ],
       [ "game_watch", 3 ],
-      [ "vibrator", 10 ],
```

Comment:
Woah. This item group is a mess lol. 

Ughh I guess feel free to make the change but I need to remove the group wholesale.